ios/Podfile

```diff
@@ -83,7 +83,7 @@ post_install do |installer|
         ## dart: PermissionGroup.criticalAlerts
         # 'PERMISSION_CRITICAL_ALERTS=1'
       ]
-      config.build_settings['IPHONE_DEPLOYMENT_TARGET']='11.0'
+      config.build_settings['IPHONE_DEPLOYMENT_TARGET']='10.0'
```

Comment:
are we sure about this? we went to 13 recently, now back to 10

Reply:
Leave it, because I'm trying to run an application in iOS.

Reply:
leave it means?